installer/templates/new/assets/javascripts/application.js

```diff
@@ -27,13 +27,35 @@ class Kitto {
     content_width = (widget_base_dimensions[0] + widget_margins[0] * 2) * columns;
 
     $gridster.width(content_width);
-    $('.gridster ul:first').gridster({
+    $('.gridster > ul').gridster({
       widget_margins: widget_margins,
       widget_base_dimensions: widget_base_dimensions
     });
 
     return this;
   }
+
+  static initializeRotator() {
+    let $rotator = $('rotator');
+    let $dashboards = $rotator.children();
+
+    if ($dashboards.length < 2) { return this; }
```

Comment:
I think we can use the presence of elements matching a selector to return early.

```
if (!$rotator) { return this };
```